core/domain/question_domain.py

```diff
@@ -253,3 +253,44 @@ def to_dict(self):
             'question_id': self.question_id,
             'question_title': self.question_title
         }
+
+
+class QuestionSkillLink(object):
+    """Domain object for Question Skill Link.
+
+    Attributes:
+        question_id: str. The ID of the question.
+        skill_id: str. The ID of the skill to which the
+                  question is linked.
+        review_status: int. Status of the review:
```

Comment:
Let's drop review_status -- @nithusha21 is working on a system that obviates it. Sorry for not mentioning this earlier.

Reply:
Done

Reply:
Please update the docstring.

Reply:
Done